tests/unit/test_metadata.py

```diff
@@ -50,32 +50,46 @@ def test_get_yaml_malformed(
     assert_substrings_in_string(expected_error_msg_contents, str(exc_info.value).lower())
 
 
+# pylint currently does not understand walrus operator perfectly yet
+# pylint: disable=unused-variable,undefined-variable
 @pytest.mark.parametrize(
-    "metadata_name, metadata_docs_url",
+    "metadata_yaml_content, expected_metadata_name, expected_metadata_docs",
```

Comment:
Would this be better if the second parameter was just `expected_metadata`?

Reply:
Changed!